pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -21,7 +21,7 @@ import (
 	"time"
 
 	"github.com/containernetworking/plugins/pkg/ip"
-	"k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Out of curiosity.. is there a rule to determine the package name as prevDirectory+currentDirectory.

Reply:
Adding this because goimports think "v1" is not a good import name because of lack of package information which I also agree. But the way it "fixes" it is adding "v1" as the import alias.
```
v1 "k8s.io/api/core/v1"
```
I'm not sure if this is a bug or a workaround that makes goimports ignore it. But I feel having a prefix can improve the readability as well as in compliance with goimports.
Both "corev1", "v1" are used in kubernetes:
https://github.com/kubernetes/kubernetes/blob/master/pkg/master/controller.go#L26
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L42

Reply:
Thanks for the explanation. corev1 sounds good.